.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libswiftnav"]
 	path = libswiftnav
-	url = https://github.com/swift-nav/libswiftnav
+	url = https://github.com/adel-mamin/libswiftnav
```

Comment:
Is this PR contingent on libswiftnav updates that haven't been updated to master? Before merging let's get the libswiftnav updates merged and point this to a commit on libswiftnav/master.